systemtest/src/test/java/io/strimzi/systemtest/watcher/AbstractNamespaceST.java

```diff
@@ -4,67 +4,326 @@
+    final void testDeployKafkaWithOperandsInNamespaceDifferentFromCO() {
+        LOGGER.info("Verifying that Kafka: {}/{} ,its Kafka Exporter and Cruise Control are in ready state", MAIN_TEST_NAMESPACE, PRIMARY_KAFKA_NAME);
+        KafkaUtils.waitForKafkaReady(MAIN_TEST_NAMESPACE, PRIMARY_KAFKA_NAME);
+        assertThat("Cruise Control deployment is not ready", kubeClient().getDeploymentStatus(MAIN_TEST_NAMESPACE, PRIMARY_KAFKA_NAME + "-cruise-control"));
+        assertThat("Kafka Exporter deployment is not ready", kubeClient().getDeploymentStatus(MAIN_TEST_NAMESPACE, PRIMARY_KAFKA_NAME + "-kafka-exporter"));
     }
 
-    void checkMirrorMakerForKafkaInDifNamespaceThanCO(ExtensionContext extensionContext, String sourceClusterName) {
-        String kafkaSourceName = sourceClusterName;
-        String kafkaTargetName = MAIN_NAMESPACE_CLUSTER_NAME + "-target";
+    /**
+     * @description This test case verifies that Cluster Operator manages Kafka Bridge deployed in watched namespace, which is different
+     * from one where Cluster Operator resides correctly.
+     *
+     * @steps
+     *  1. - KafkaBridge custom resource is deployed in namespace watched by Cluster Operator.
+     *     - Kafka Bridge is transitioned into ready state.
+     *
+     * @usecase
+     *  - namespaces
+     *  - cluster-operator-watcher
+     *  - bridge
+     */
+    @IsolatedTest
+    final void testKafkaBridgeInDifferentNamespaceFromCO(ExtensionContext extensionContext) {
+        TestStorage testStorage = new TestStorage(extensionContext, MAIN_TEST_NAMESPACE);
+        String bridgeName = testStorage.getClusterName() + "-bridge";
 
-        String previousNamespace = cluster.setNamespace(SECOND_NAMESPACE);
-        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(kafkaTargetName, 1, 1).build());
-        resourceManager.createResource(extensionContext, KafkaMirrorMakerTemplates.kafkaMirrorMaker(MAIN_NAMESPACE_CLUSTER_NAME, kafkaSourceName, kafkaTargetName, "my-group", 1, false).build());
+        LOGGER.info("Creating KafkaBridge: {}/{} in namespace different from Cluster Operator's", MAIN_TEST_NAMESPACE, bridgeName);
+        resourceManager.createResource(extensionContext, KafkaBridgeTemplates.kafkaBridge(bridgeName,
+                KafkaResources.plainBootstrapAddress(PRIMARY_KAFKA_NAME), 1)
+            .editMetadata()
+                .withNamespace(testStorage.getNamespaceName())
+            .endMetadata()
+            .build()
+        );
 
-        LOGGER.info("Waiting for creation {}/{}", SECOND_NAMESPACE, MAIN_NAMESPACE_CLUSTER_NAME + "-mirror-maker");
-        KafkaMirrorMakerUtils.waitForKafkaMirrorMakerReady(SECOND_NAMESPACE, MAIN_NAMESPACE_CLUSTER_NAME);
-        cluster.setNamespace(previousNamespace);
+        LOGGER.info("Verifying KafkaBridge: {}/{} transition into ready state", MAIN_TEST_NAMESPACE, bridgeName);
+        KafkaBridgeUtils.waitForKafkaBridgeReady(testStorage.getNamespaceName(), bridgeName);
```

Comment:
You are waiting for the same thing inside the `createResource`, so I would maybe remove this or add some other check